{"uuid":"298d6663-5c54-4de5-8e39-0eb259f9cf87","historyId":"76c95338467c5540255bc7514e33b9a0","fullName":"cases.login.AuthLoginTest.loginTest","labels":[{"name":"package","value":"cases.login.AuthLoginTest"},{"name":"testClass","value":"cases.login.AuthLoginTest"},{"name":"testMethod","value":"loginTest"},{"name":"parentSuite","value":"TeacherTest"},{"name":"suite","value":"login"},{"name":"subSuite","value":"cases.login.AuthLoginTest"},{"name":"host","value":"TiandeMacBook-Pro.local"},{"name":"thread","value":"56712@TiandeMacBook-Pro.local.main(1)"}],"links":[],"name":"教师登录","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.simfast.simpletester.exception.SimpleTesterException: The data file is not found! File is: classpath:testData/AuthLoginTestData.xls","trace":"java.lang.RuntimeException: org.simfast.simpletester.exception.SimpleTesterException: The data file is not found! File is: classpath:testData/AuthLoginTestData.xls\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)\n\tat org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:131)\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:706)\n\tat org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)\n\tat org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:73)\n\tat org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)\nCaused by: org.simfast.simpletester.exception.SimpleTesterException: The data file is not found! File is: classpath:testData/AuthLoginTestData.xls\n\tat org.simfast.simpletester.core.DataResoverUtils.resolveExcelTestParams(DataResoverUtils.java:101)\n\tat cases.login.AuthLoginTest.setupExcel(AuthLoginTest.java:42)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)\n\t... 21 more\nCaused by: java.io.FileNotFoundException: class path resource [testData/AuthLoginTestData.xls] cannot be resolved to absolute file path because it does not exist\n\tat org.springframework.util.ResourceUtils.getFile(ResourceUtils.java:178)\n\tat org.simfast.simpletester.core.DataResoverUtils.resolveExcelTestParams(DataResoverUtils.java:91)\n\t... 29 more\n"},"stage":"finished","start":1603547145236,"stop":1603547145237,"parameters":[]}